hack/olm-registry/olm-artifacts-template.yaml

```diff
@@ -108,6 +108,14 @@ objects:
             - get
             - list
             - watch
+          - apiGroups:
+            - ""
+            resources:
+            - serviceaccounts/token
+            verbs:
+            - create
+            - update
+            - patch
```

Comment:
Just wanted to confirm if we need the patch and update verbs as well?

Reply:
Removed unnecessary verbs